snuba/migrations/snuba_migrations/sessions/0001_sessions.py

```diff
@@ -137,7 +87,7 @@ def forwards_dist(self) -> Sequence[operations.Operation]:
             operations.CreateTable(
                 storage_set=StorageSetKey.SESSIONS,
                 table_name="sessions_hourly_dist",
-                columns=aggregate_columns,
+                columns=aggregate_columns_v1,
```

Comment:
Not sure i get the reason for moving the aggregated columns and the materialized view statement out of the migration file, they should not be reused anywhere. Since the migrations must be semantically immutable, it is generally safe to keep refactoring them to a minimum unless strictly necessary.